docs/source/support/release-notes.rst

```diff
@@ -6,6 +6,304 @@ Release notes
 
 This page contains release notes for the SDK.
 
+.. _release-1.4.0:
+
+1.4.0 - 2020-08-12
+------------------
+
+Highlights
+~~~~~~~~~~
+
+- DAML on SQL is available as a standalone JAR, `you can find it on
```

Comment:
This is missing the most recent changes from doc and blog. We are only releasing as Early Access and not advising migration yat.

Reply:
Updated, the changes were missing from the blog as well which has now been updated.